network-formula/network/wicked/service.sls

```diff
@@ -0,0 +1,28 @@
+{#-
+Salt state file for managing the Wicked network service
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'network/wicked/map.jinja' import script -%}
```

Comment:
Unused import

Reply:
Nice but unfortunate catch, I liked the round 800 in the PR line count.

Reply:
You could add some more comments explaining non-obvious lines ;-)

Reply:
;-) I did add some more comments to the script.